package.json

```diff
@@ -124,10 +124,10 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^27.3.1",
     "jsdom": "^18.1.1",
-    "lint-staged": "12.1.1",
-    "prettier": "2.4.1",
+    "lint-staged": "12.1.2",
+    "prettier": "2.5.0",
     "serve": "^13.0.2",
-    "storybook-addon-gatsby": "^0.0.2",
+    "storybook-addon-gatsby": "^0.0.3",
```

Comment:
Seeing such a early version updated gives me an uncanny feeling, so I checked. Looks like a patch that deserves the name patch. Very small updates regarding the CLI and the peer dependency to support gatsby 3 & 4. 

https://github.com/prismicio-community/storybook-addon-gatsby/blob/20754782615a09b6fc50b3026d04234027f7b022/CHANGELOG.md